packages/vega4-extension/src/index.ts

```diff
@@ -153,3 +157,46 @@ const extension: IRenderMime.IExtension = {
 };
 
 export default extension;
+
+/**
+ * A namespace for private module data.
+ */
+namespace Private {
```

Comment:
This seemed like the most common pattern for weird global stuff... thoughts?

Reply:
👍🏼